### PR TITLE
key_value_store: fix missing std

### PR DESCRIPTION
**Pull request by ronen-fr**
required in order to compile some scrub code, following PR #42742

Signed-off-by: Ronen Friedman <rfriedma@redhat.com>

### CODE REVIEW

File: src/key_value_store/kv_flat_btree_async.h [hunk 1]
```diff
@@ -33,6 +33,13 @@
 #include <stdarg.h>
 
 using ceph::bufferlist;
+using std::pair;
+using std::make_pair;
+using std::cout;
+using std::vector;
+using std::cerr;
+using std::stringstream;
```
Comment (tchaikov) [inline src/key_value_store/kv_flat_btree_async.h:41]: can we avoid using `using ...` in the global naming space?

Comment (ronen-fr) [reply src/key_value_store/kv_flat_btree_async.h:41]: OK. Will redo.

**Review by tchaikov — APPROVED** [review]

**Comment by ronen-fr** [comment]
@tchaikov : sorry - an extra change is needed.

Your opinion: as this is a compilation fix - OK to merge without formal testing? Can't rebase my other PRs without
this merged.

**Comment by tchaikov** [comment]
@ronen-fr `WITH_KVS` is OFF by default. so this piece of code is not tested in our CI, so i am fine either way.

**Review by tchaikov — APPROVED** [review]

**Pull request merged**